frontend/src/constants/localStorage.ts

```diff
@@ -19,4 +19,6 @@ export enum LOCALSTORAGE {
 	SHOW_EXPLORER_TOOLBAR = 'SHOW_EXPLORER_TOOLBAR',
 	PINNED_ATTRIBUTES = 'PINNED_ATTRIBUTES',
 	THEME_ANALYTICS_V1 = 'THEME_ANALYTICS_V1',
+	LAST_USED_SAVED_LOGS_VIEW = 'LAST_USED_SAVED_LOGS_VIEW',
```

Comment:
rather than 2 different keys, we can have `LAST_USED_SAVED_VIEWS` : { logs : {}, traces: {}`

Reply:
Done, @vikrantgupta25 :slightly_smiling_face: